server/services/waypoint.js

```diff
@@ -375,6 +391,16 @@ module.exports = class WaypointService {
         }
     }
 
+    _performWaypointActionCollectPercentage(carrier, star, waypoint) {
+        const toTransfer = Math.floor(star.garrison * (waypoint.actionShips * 0.01))
+
+        if (toTransfer > 1 && star.garrison - toTransfer >= 0) {
```

Comment:
I think this `toTransfer > 1` should be `toTransfer >= 1`? Stars can have 0 ships.

Reply:
Agreed, it seems that I missed that.